src/components/createPosts.js

```diff
@@ -0,0 +1,144 @@
+import React, {useState, useContext} from 'react';
+import {
+  Text,
+  Modal,
+  View,
+  StyleSheet,
+  TouchableWithoutFeedback,
+  Keyboard,
+} from 'react-native';
+import InputText from './inputText';
+import Button from './button';
+import AppContext from '../useContext/AppContext';
+import ShowError from './showError';
+import validate from './validator';
+import {showToast} from '../utils/LocalStorage';
+import {postmethod} from '../utils/LocalStorage';
+
+const CreatePosts = ({visible, handleModal, getAllPosts}) => {
```

Comment:
@Charan017 ~CreatePosts~ -> `CreatePostModal` for the purpose clarity.

Reply:
Done